_observing-your-data/alerting/monitors.md

```diff
@@ -10,348 +10,24 @@ redirect_from:
 
 # Monitors
 
---- 
-
-<details closed markdown="block">
-  <summary>
-    Table of contents
-  </summary>
-  {: .text-delta }
-- TOC
-{:toc}
-</details>
-
----
+Proactively monitor your data in OpenSearch with alerting and anomaly detection. Set up alerts to receive notifications when your data exceeds certain thresholds. Anomaly detection uses machine learning (ML) to automatically detect any outliers in your streaming data. You can pair anomaly detection with alerting to ensure that you're notified as soon as an anomaly is detected.
```

Comment:
If "alerting and anomaly detection" are in reference to the plugins (or, extension, now in the case of Anomaly Detection), I believe these mentions should be capitalized.